tests/routes/test_peers.py

```
@@ -561,8 +562,72 @@
 
     response = client.post(
         f"/v2/workspaces/{test_workspace.name}/peers/{nonexistent_peer_id}/search",
-        json="test query",
+        json={"query": "test query", "use_semantic_search": False},
     )
     # This should probably return 404 or handle gracefully
     # The exact behavior depends on the crud.search implementation
     assert response.status_code in [200, 404, 422]
+
+
+def test_search_peer_with_semantic_search_false(
```

Ruff (0.11.9)

Missing return type annotation for public function `test_search_peer_with_semantic_search_false`

Add return type annotation: `None`

(ANN201)